src/main/java/org/jenkinsci/remoting/engine/HostPort.java

```diff
@@ -1,28 +1,38 @@
 package org.jenkinsci.remoting.engine;
 
 class HostPort {
+
+    private static final int PORT_MIN = 0;
+    private static final int PORT_MAX = (1 << 16) -1;
```

Comment:
```suggestion
    private static final int PORT_MAX = (1 << 16) - 1;
```

Otherwise looks like multiplication.

Reply:
That's a good point.